ConsoleGUI/Controls/VerticalScrollPanel.cs

```diff
@@ -39,6 +42,10 @@ public int Top
 		}
 
 		private Character _scrollBarForeground = new Character('▀', foreground: new Color(100, 100, 255));
+
```

Comment:
Yup, same comment as before, but here please also make sure that those two properties are not squeezed in between the `ScrollBarForeground` property and its backing field :P